scripts/parsers/cohere_jobs.py

```
@@ -0,0 +1,132 @@
+    "Product",
+}
+
+
+def ashby_api_url(source_url: str) -> str:
+    parsed = urlparse(source_url)
+    if parsed.netloc == "api.ashbyhq.com":
+        return source_url
+
+    board_slug = parsed.path.strip("/").split("/")[0]
+    if not board_slug:
+        raise ValueError(f"Cannot infer Ashby board slug from {source_url}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)